ding/utils/log_writer_helper.py

```diff
@@ -0,0 +1,97 @@
+from tensorboardX import SummaryWriter
+from typing import TYPE_CHECKING
+if TYPE_CHECKING:
```

Comment:
add some comments for why use this `TYPE_CHECKING`, I guess it's because of circle import

Reply:
Done